recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
I guess I'll stop being lazy, fix the autotools script, put it as a patch here, and send a PR upstream, because why not fix it for everyone....